### PR TITLE
Choose and update rest of PF2 exercises

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

**Comment by KristinaRiemer** [comment]
I forgot to add in the option to use apply instead of a for loop in the last exercise, so don't merge this yet.

**Comment by ethanwhite** [comment]
OK, just ping me when it's ready for review.

**Comment by KristinaRiemer** [comment]
@ethanwhite done. Good thing I did that, my first solution was wrong!

File: exercises/Loops-multiple-files-R.md [hunk 1]
```diff
@@ -7,49 +7,37 @@ language: R
...
-*Optional*: For a little extra challenge change your answer so that instead of
-printing out the file names it prints out the species name that is encoded in
-the file name without the `".data/archaea-dna/"` path at the beginning and the
-`".fasta"` extension at the end.
+Use `list.files()`, with `full.names` set to true, to generate a list of the names
+of all the sequence files. Then create a for loop that uses the above `seq` code to 
+read in each sequence file. When this works, add to the loop a function that 
+calculates GC content of each file. You can use the GC content function you wrote for 
+[stringr]({{site.baseurl}}/exercises/Loops-stringr-R), but it will have to be 
+modified due to capitalization in the sequence files. Once this is working, create
```
Comment (ethanwhite) [inline exercises/Loops-multiple-files-R.md]: Maybe tell them how to make this modification?

File: exercises/Loops-multiple-files-R.md [hunk 2]
```diff
@@ -7,49 +7,37 @@ language: R
...
+an empty data frame that is filled with values from the for loop, one column with
```
Comment (ethanwhite) [inline exercises/Loops-multiple-files-R.md]: "an empty data frame to store the output and fill it with values from..."

File: exercises/Loops-stringr-R.md [hunk 3]
```diff
@@ -6,49 +6,36 @@ language: R
 ---
 
 A colleague has produced a file with one DNA sequence on each line. Download
-[the file]({{ site.baseurl }}/data/dna-sequences-1.txt) and load it into R using 
-`read.csv()`. The file has no header. Name the resulting data frame `sequences`.
+[the file]({{ site.baseurl }}/data/dna-sequences-1.txt). Load it into R with`read.table()`, setting `header = TRUE` and naming the resulting data frame `sequences`.
```
Comment (ethanwhite) [inline exercises/Loops-stringr-R.md]: Any reason not to use `read.csv()` which they are used to and assumes the existence of the header?

Comment (KristinaRiemer) [reply exercises/Loops-stringr-R.md]: Duh.

**Comment by ethanwhite** [comment]
:+1:

**Pull request merged**